### ISSUE TITLE
Does this work with Confluent Plattform 4.x ?

### ISSUE THREAD

**Issue by wintersommer** [Landoop/stream-reactor#339]
I just build with gradle and saw that it pulled dependencies from 3.3.0. Did i see that correctly ? Does that work with the 4.0.0 Kafka Connect ?

**Comment by stheppi** [comment]
Hi @wintersommer, 
  We haven't released for 1.0 and confluent 4.0 yet.  It will happen later this month. 
  If you would look at Release Notes you would notice: 3.0.1 is pending and 3.0.0 says

```3.0.0

    Upgrade CoAP to 2.0.0-M4
    Upgrade to Confluent 3.3 and Kafka 0.11.0.0.
```
To answer your question about working on 4.0.0, you will have to find that yourself I am affraid. We don't intend to cover such testing.

**Comment by Antwnis** [comment]
It's high on the priority list to make a new release for Kafka 1.0 @wintersommer - so it's coming soon :)

If you give it a go, please let us know if you make your own build by updating the relevant libs
